src/pybind/mgr/dashboard/frontend/src/app/core/error/error.component.html

```diff
@@ -59,9 +59,19 @@ <h3 i18n><b>Page not Found</b></h3>
 </ng-template>
 
 <ng-template #dashboardButton>
-  <div class="mt-4 text-center">
+  <div class="mt-4 text-center"
+       *ngIf="!buttonEnable; else enableButton">
     <button class="btn btn-primary"
             [routerLink]="'/dashboard'"
             i18n>Go To Dashboard</button>
   </div>
 </ng-template>
+
+<ng-template #enableButton>
+  <div class="mt-4 text-center"
+       *ngIf="buttonEnable && !(buttonName && buttonRoute)">
+    <button class="btn btn-primary"
+            (click)="enableModule()"
+            i18n>Enable</button>
```

Comment:
more explicit name if we want to
```suggestion
            i18n>Enable {{ moduleToEnable | upperCase }} module</button>
```

Reply:
I was making this change, but for some reason the upperCase pipe was failint to load and crashing with:  
```
zone.js:218 Uncaught RuntimeError: NG0302: The pipe 'upperCase' could not be found in the 'ErrorComponent' component. Verify that it is declared or imported in this module. Find more at https://angular.io/errors/NG0302
    at getPipeDef (core.mjs:20764:15)
    at Module.ɵɵpipe (core.mjs:20720:19)
    at ErrorComponent_ng_template_16_div_0_Template (error.component.html:76:18)
    at executeTemplate (core.mjs:10441:9)
    at renderView (core.mjs:10263:13)
    at TemplateRef.createEmbeddedView (core.mjs:21143:9)
    at ViewContainerRef.createEmbeddedView (core.mjs:21254:37)
    at NgIf._updateView (common.mjs:3344:45)
    at set ngIf (common.mjs:3317:14)
    at setInputsForProperty (core.mjs:11646:34)
   ```
   I can see the `CommonModule` imported in the [core.module.ts file](https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/core/core.module.ts) where `ErrorComponent` is defined, what am I missing here?
   
  Anyways. for now I'm using `upperFirst` pipe

Reply:
ah my bad, it might have been called as `uppercase`.
https://github.com/angular/angular/blob/main/packages/common/src/pipes/case_conversion_pipes.ts#L108